ocp-templates/mpp/frontend.yaml

```diff
@@ -144,6 +163,10 @@ parameters:
   displayName: Frontend Route
   description: The URL of the frontend of the Ibutsu server
   value: ibutsu.example.com
+- name: FRONTEND_ROUTE_CNAME
+  displayName: Frontend Route
+  description: The CNAME URL of the frontend of the Ibutsu server
+  value: ibutsu.prod.com
```

Comment:
Why do we need separate route parameters ? can't we use just 1 route, set the value at creation time with the CNAME URL ?